main/survey/survey.lib.php

```diff
@@ -2877,6 +2897,7 @@ function jumpMenu(targ,selObj,restore) {
      * @todo the pagebreak and comment question types should not be shown => removed from $survey_data before
      * @author Patrick Cool <patrick.cool@UGent.be>, Ghent University
      * @version February 2007 - Updated March 2008
+     * @CSS Alex Aragon <alex.aragon@beeznest.com> - update January 2015
```

Comment:
This is not a recognized phpdoc tag. Additionally, a function should only have an @author tag if it is very important to be able to identify the author (for example, it is a very complex function which depends on stuff that we are not sure about). Otherwise, no @author tag, and certainly not just for CSS changes (which should be located outside of a php script anyway)